handout/features/spread_and_rest.md

```diff
@@ -1,14 +1,14 @@
-# Operators Spread and Rest
+# Spread Syntax (Spread Element) and Rest parameters
 
-A Spread operator allows in-place expansion of an expression for the following cases:
+A Spread syntax allows in-place expansion of an expression for the following cases:
```

Comment:
I guess `...` is [not explicitly an operator](http://www.ecma-international.org/ecma-262/6.0/ "ES 6.0 Spec") although I don't feel like calling it a syntax is quite correct either.  The official term "Punctuator" also seems awkward @barretodavid any thoughts on this?